packages/lambda-tiler/src/util/config.cache.ts

```diff
@@ -0,0 +1,40 @@
+import { ConfigBundled, ConfigProviderMemory } from '@basemaps/config';
+import { fsa } from '@chunkd/fs';
+import { SwappingLru } from './swapping.lru.js';
+
+class LruConfig {
+  value: ConfigProviderMemory;
+
+  constructor(mem: ConfigProviderMemory) {
+    this.value = mem;
+  }
+
+  get size(): number {
+    return JSON.stringify(this.value.toJson()).length;
```

Comment:
this will be super slow, your best bet is to do it once in the constructor. but it would be even faster to just jut `fsa.read()` and get the length of the bufffer rather than the `readJson`